test/index.js

```diff
@@ -1,53 +1,68 @@
 const path = require('path')
 const tape = require('tape')
 
-const { Diorama, tapeExecutor, backwardCompatibilityMiddleware } = require('@holochain/diorama')
+const { Orchestrator, Config, tapeExecutor } = require('@holochain/try-o-rama')
 
 process.on('unhandledRejection', error => {
   // Will print "unhandledRejection err is not defined"
   console.error('got unhandledRejection:', error);
 });
 
 const dnaPath = path.join(__dirname, "../dist/passthrough-dna.dna.json")
-const dna = Diorama.dna(dnaPath, 'passthrough-dna')
-
-const diorama = new Diorama({
-  instances: {
-    alice: dna,
-    bob: dna,
-  },
-  bridges: [],
-  debugLog: false,
-  executor: tapeExecutor(require('tape')),
-  middleware: backwardCompatibilityMiddleware,
+const dna = Config.dna(dnaPath, 'passthrough-dna')
+const config = {
+  instances: { app: dna }
+}
+
+var transport_config = {
+    type: 'sim1h',
+    dynamo_url: "http://localhost:8000"
+}
```

Comment:
Hey @zippy maybe we should leave the default config to be n3h here?

Reply:
um, I thought it wasn't working?

Reply:
@zippy it mostly does, but sim1h doesn't work at all if the user doesn't know how to spin up a local db instance. And the type of user who would know how to do that would be providing their own networking config.